framework/src/media/Decoder.cpp

```diff
@@ -135,8 +135,8 @@ bool Decoder::mConfig(int audioType)
 		mp3_ext.pOutputBuffer = outputBuf;
 		mp3_ext.outputFrameSize = sizeof(outputBuf) / sizeof(int16_t);
 
-		if (audio_decoder_init_decoder(&mDecoder, audioType, &mp3_ext) != AUDIO_DECODER_OK) {
-			meddbg("Error! audio_decoder_init_decoder failed!\n");
+		if (audio_decoder_config_decoder(&mDecoder, audioType, &mp3_ext) != AUDIO_DECODER_OK) {
```

Comment:
How about `audio_decoder_configure`?
duplicated `decoder` seems not good, and function name should be verb